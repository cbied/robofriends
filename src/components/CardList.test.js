import React from 'react'
import Cardlist from './CardList'
import { shallow, configure } from 'enzyme'
import Adapter from '@cfaester/enzyme-adapter-react-18';

configure({adapter: new Adapter()});

const filteredRobots = [
    {
        id: 1,
        name: 'Mom',
        username: 'yourMom',
        email: 'yourMom@gmail.com'
    }
]

it('should render without crashing', () => {
    expect(shallow(<Cardlist robots={filteredRobots}/>)).toMatchSnapshot()
})