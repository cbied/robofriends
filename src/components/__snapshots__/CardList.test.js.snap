// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render without crashing 1`] = `
<div>
  <Card
    email="yourMom@gmail.com"
    id={1}
    key="0"
    name="Mom"
  />
</div>
`;
